### PR TITLE
Make documentation mobile friendly(ish)

### PR DESCRIPTION
**Pull request by mwhipple**
A minimal patch to the documentation so that content isn't hidden and more minimal on smaller screens (mobile).  Display on larger screens remain as they were.

### CODE REVIEW

File: logback-site/src/site/pages/manual/menu.js [hunk 1]
```diff
@@ -1,5 +1,15 @@
+(function() {
```
Comment (tony19) [inline logback-site/src/site/pages/manual/menu.js]: Codacy Issue found: [Move the invocation into the parens that contain the function.](https://www.codacy.com/app/tony19/logback/file/2375991541/issues/source?bid=3222618&fileBranchId=3222700#l1)

File: logback-site/src/site/pages/css/screen.css [hunk 2]
```diff
@@ -217,4 +217,54 @@ td:hover .anchor {
...
+  }
+  #right:target+#right_open::after,#left:target+#left_open::after {
+    content: '';
+  }
+  .handle {
+    display: inline-block;
+    width: 50%;
+    height: 1.5em;
+    background-color: #FFE0B0;
+    border: thin solid #cccccc;
```
Comment (tony19) [inline logback-site/src/site/pages/css/screen.css]: Codacy Issue found: [Using height with border can sometimes make elements larger than you expect.](https://www.codacy.com/app/tony19/logback/file/2375991531/issues/source?bid=3222618&fileBranchId=3222700#l243)

File: logback-site/src/site/pages/manual/menu.js [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+  var rightEl = document.getElementById('right');
+  var rightOpenEl = document.createElement('a');
+  rightOpenEl.href = '#right';
+  rightOpenEl.className = 'handle right_handle';
+  rightOpenEl.id = 'right_open';
+  rightOpenEl.innerHTML = 'Chapter Index'
```
Comment (tony19) [inline logback-site/src/site/pages/manual/menu.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/tony19/logback/file/2375991541/issues/source?bid=3222618&fileBranchId=3222700#l7)

File: logback-site/src/site/pages/manual/menu.js [hunk 4]
```diff
@@ -1,5 +1,15 @@
+(function() {
+  var rightEl = document.getElementById('right');
+  var rightOpenEl = document.createElement('a');
+  rightOpenEl.href = '#right';
```
Comment (tony19) [inline logback-site/src/site/pages/manual/menu.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/tony19/logback/file/2375991541/issues/source?bid=3222618&fileBranchId=3222700#l4)

File: logback-site/src/site/pages/manual/menu.js (hunk 3 above)
Comment (tony19) [inline logback-site/src/site/pages/manual/menu.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/tony19/logback/file/2375991541/issues/source?bid=3222618&fileBranchId=3222700#l7)

File: logback-site/src/site/pages/manual/menu.js [hunk 5]
```diff
@@ -1,5 +1,15 @@
...
+  rightOpenEl.className = 'handle right_handle';
+  rightOpenEl.id = 'right_open';
+  rightOpenEl.innerHTML = 'Chapter Index'
+  rightEl.parentNode.insertBefore(rightOpenEl, rightEl.nextSibling);
+})();
+document.write('<a href="#right_close" class="slide_close" id="right_close"></a>');
```
Comment (tony19) [inline logback-site/src/site/pages/manual/menu.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/tony19/logback/file/2375991541/issues/source?bid=3222618&fileBranchId=3222700#l10)

File: logback-site/src/site/pages/manual/menu.js [hunk 6]
```diff
@@ -1,5 +1,15 @@
+(function() {
+  var rightEl = document.getElementById('right');
+  var rightOpenEl = document.createElement('a');
+  rightOpenEl.href = '#right';
+  rightOpenEl.className = 'handle right_handle';
```
Comment (tony19) [inline logback-site/src/site/pages/manual/menu.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/tony19/logback/file/2375991541/issues/source?bid=3222618&fileBranchId=3222700#l5)

File: logback-site/src/site/pages/manual/menu.js [hunk 7]
```diff
@@ -1,5 +1,15 @@
...
+  rightOpenEl.id = 'right_open';
```
Comment (tony19) [inline logback-site/src/site/pages/manual/menu.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/tony19/logback/file/2375991541/issues/source?bid=3222618&fileBranchId=3222700#l6)

File: logback-site/src/site/pages/manual/menu.js [hunk 8]
```diff
@@ -1,5 +1,15 @@
...
+  rightOpenEl.innerHTML = 'Chapter Index'
+  rightEl.parentNode.insertBefore(rightOpenEl, rightEl.nextSibling);
+})();
+document.write('<a href="#right_close" class="slide_close" id="right_close"></a>');
 
-document.write('<p class="menu_header">Chapter Index</p>')
+document.write('<p class="menu_header">Chapter Index</p>');
```
Comment (tony19) [inline logback-site/src/site/pages/manual/menu.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/tony19/logback/file/2375991541/issues/source?bid=3222618&fileBranchId=3222700#l12)

File: logback-site/src/site/pages/manual/menu.js [hunk 9]
```diff
@@ -1,5 +1,15 @@
+(function() {
+  var rightEl = document.getElementById('right');
+  var rightOpenEl = document.createElement('a');
```
Comment (tony19) [inline logback-site/src/site/pages/manual/menu.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/tony19/logback/file/2375991541/issues/source?bid=3222618&fileBranchId=3222700#l3)

File: logback-site/src/site/pages/css/screen.css (hunk 2 above)
Comment (tony19) [inline logback-site/src/site/pages/css/screen.css]: Codacy Issue found: [Using height with border can sometimes make elements larger than you expect.](https://www.codacy.com/app/tony19/logback/file/2375991531/issues/source?bid=3222618&fileBranchId=3222700#l243)

File: logback-site/src/site/pages/manual/menu.js (hunk 1 above)
Comment (tony19) [inline logback-site/src/site/pages/manual/menu.js]: Codacy Issue found: [Move the invocation into the parens that contain the function.](https://www.codacy.com/app/tony19/logback/file/2375991541/issues/source?bid=3222618&fileBranchId=3222700#l1)

File: logback-site/src/site/pages/manual/menu.js (hunk 3 above)
Comment (tony19) [inline logback-site/src/site/pages/manual/menu.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/tony19/logback/file/2375991541/issues/source?bid=3222618&fileBranchId=3222700#l7)

File: logback-site/src/site/pages/manual/menu.js (hunk 4 above)
Comment (tony19) [inline logback-site/src/site/pages/manual/menu.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/tony19/logback/file/2375991541/issues/source?bid=3222618&fileBranchId=3222700#l4)

File: logback-site/src/site/pages/manual/menu.js (hunk 3 above)
Comment (tony19) [inline logback-site/src/site/pages/manual/menu.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/tony19/logback/file/2375991541/issues/source?bid=3222618&fileBranchId=3222700#l7)

File: logback-site/src/site/pages/manual/menu.js (hunk 5 above)
Comment (tony19) [inline logback-site/src/site/pages/manual/menu.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/tony19/logback/file/2375991541/issues/source?bid=3222618&fileBranchId=3222700#l10)

File: logback-site/src/site/pages/manual/menu.js (hunk 6 above)
Comment (tony19) [inline logback-site/src/site/pages/manual/menu.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/tony19/logback/file/2375991541/issues/source?bid=3222618&fileBranchId=3222700#l5)

File: logback-site/src/site/pages/manual/menu.js (hunk 7 above)
Comment (tony19) [inline logback-site/src/site/pages/manual/menu.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/tony19/logback/file/2375991541/issues/source?bid=3222618&fileBranchId=3222700#l6)

File: logback-site/src/site/pages/manual/menu.js (hunk 8 above)
Comment (tony19) [inline logback-site/src/site/pages/manual/menu.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/tony19/logback/file/2375991541/issues/source?bid=3222618&fileBranchId=3222700#l12)

File: logback-site/src/site/pages/manual/menu.js (hunk 9 above)
Comment (tony19) [inline logback-site/src/site/pages/manual/menu.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/tony19/logback/file/2375991541/issues/source?bid=3222618&fileBranchId=3222700#l3)

**Comment by mwhipple** [comment]
I'm closing this since it was meant to be a quick patch and is now very stale

**Pull request closed without merging**